imagetest/testworkflow.go

```diff
@@ -48,12 +48,119 @@ type TestWorkflow struct {
 	// ShortImage will be only the final component of Image, used for naming.
 	ShortImage string
 	// destination for workflow outputs in GCS.
-	destination    string
+	gcsPath        string
 	skipped        bool
 	skippedMessage string
 	wf             *daisy.Workflow
 }
 
+func (t *TestWorkflow) appendCreateVMStep(name string) (*daisy.Step, error) {
+	attachedDisk := &compute.AttachedDisk{Source: name}
+
+	instance := &daisy.Instance{}
+	instance.StartupScript = "wrapper"
+	instance.Name = name
+	instance.Scopes = append(instance.Scopes, "https://www.googleapis.com/auth/devstorage.read_write")
+	instance.Disks = append(instance.Disks, attachedDisk)
+
+	instance.Metadata = make(map[string]string)
+	instance.Metadata["_test_vmname"] = name
+	instance.Metadata["_test_package_url"] = "${SOURCESPATH}/testpackage"
+	instance.Metadata["_test_results_url"] = fmt.Sprintf("${OUTSPATH}/%s.txt", name)
```

Comment:
Please use a composite literal declaration to initialize instance all at once. (see go/go-style/best-practices#vardeclcomposite)

Reply:
not for daisy, daisy has a bunch of weird types that make us do this. it's also a heavily nested structure,  so this style becomes more readable in this instance

Reply:
```suggestion
	instance := &daisy.Instance{
		StartupScript: "wrapper",
		Name:          name,
		Scopes:       []string{"https://www.googleapis.com/auth/devstorage.read_write"},
		Disks:        []disk{attachedDisk},
		Metadata:      map[string]string{
			"_test_vmname": name,
			"_test_package_url": "${SOURCESPATH}/testpackage",
			"_test_results_url": fmt.Sprintf("${OUTSPATH}/%s.txt", name),
		}
	}
```